File: Data Structures & Algorithms/sliding-window-maximum/submission-2.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
        l = 0
        r = k
        res = []
        for i in range(r, len(nums) + 1, 1):
            res.append(max(nums[l:i]))
            l += 1
        return res

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-2.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {
        vector<int> res;
        for (int i = 0; i < nums.size(); i++) {
            res.push_back(1);
            for (int j = 0; j < nums.size(); j++) {
                if (j != i) {
                    res[i] *= nums[j];
                }
            }
        }
        return res;
    }
};
